angular/package.json

```diff
@@ -19,6 +19,7 @@
     "@angular/forms": "^14.2.12",
     "@angular/platform-browser": "^14.2.12",
     "@angular/platform-browser-dynamic": "^14.2.12",
+    "@haiilo/catalyst-icons": "0.3.23",
```

Comment:
yes but there is no package.json for demo. TBH i do not understand the setup completely 😆

Reply:
Its plain Angular CLI, but I also don't fully get why there is no package.json in the demo project. Just want to avoid, that this end up in the bundle. But thats not the case. so all good